autogen/main/variables.tf.tmpl

```diff
@@ -838,6 +838,12 @@ variable "stateful_ha" {
   default     = false
 }
 
+variable "parallelstore_csi_driver" {
+  type        = bool
+  description = "Whether the Parallelstore CSI driver Addon is enabled for this cluster."
+  default     = false
```

Comment:
As there is a dynamic block using a `for_each`, could the default be made `null` so this is optional?

Reply:
will check